Formula/fcl.rb

```diff
@@ -0,0 +1,30 @@
+class Fcl < Formula
+  desc "Flexible Collision Library"
+  homepage "https://flexible-collision-library.github.io/"
+  url "https://github.com/flexible-collision-library/fcl/archive/0.5.0.tar.gz"
+  sha256 "8e6c19720e77024c1fbff5a912d81e8f28004208864607447bc90a31f18fb41a"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "flann"
```

Comment:
Could you remove `flann`, which is no longer the dependency of FCL since v0.5 (by https://github.com/flexible-collision-library/fcl/commit/cf12686bb295278b8995ba6e4da5ea38ddfba894).